tests/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -672,3 +673,29 @@ def val_dataloader(self):
         enable_model_summary=False,
     )
     trainer.fit(model)
+
+
+@pytest.mark.parametrize("num_dataloaders", [1, 2, 11])
```

Comment:
do we need to test 11? test runtime is important

Reply:
Hi Carlos. Yes we need to test 11 because the double-digit was what highlighted the bug in the first place.